index.html

```diff
@@ -4,430 +4,336 @@
-            built or a project that needs coding. I’d love to help with it  </span>
-        </div> <br>
-        <div class="contact-item-3">  
-          <form action="https://formspree.io/f/mqknglqn" method="post">
-            <input class="form" type="text" name="username" maxlength="30" placeholder="Your name" required> <br>
-            <input class="form" type="email" name="useremail" placeholder="Enter Email" required> <br>
-            <textarea class="form" cols="30" name="description" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>   <br>    
-            <button class="form-btn" type="submit">Get it Touch</button> <br>
-        </form>
-        </div>
-      </div>
-    </footer>
-  </div>
-
-  <div class="mobile">
-    <div class="container">
-      <article>
-        <header>
-          <a class="logo" href="#">KIBOMA</a>
-          <a href="#"><img src="imgs/showhide_buttons.png" alt="" /></a>
-        </header>
-      
-        <br><br><br><br><br><br>
-
-        <h1>I’m Omayio Glad to see you!</h1>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <h5>LET’S CONNECT</h5>
-        <ul class="flex">
-          <li><a href="#"><img src="imgs/Twitter.png" alt="Twitter Icon" /></a></li>
-          <li><a href="#"><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></a></li>
-          <li><a href="#"><img src="imgs/M.png" alt="M icon" /></a></li>
-          <li><a href="#"><img src="imgs/Github.png" alt="Github Icon" /></a></li>
-          <li><a href="#"><img src="imgs/hands.png" alt="hands" /></a></li>
-        </ul><br><br><br><br><br><br><br>
-      </article>
-
-      <section class="grid-container">
-        <div class="grid-item show-case">
-          <div class="content">
-            <h4>Tonic</h4>
-            <ul class="flex">
-              <li class="cano">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="flex ocontent">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-          <button type="submit">See Project</button>
-        </div>
-        <div class="grid-item show-case2">
-          <div class="content">
-            <h4>Multi-Post Stories</h4>
-            <ul class="flex">
-              <li class="cano">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
+        <div class="contact-item-2">
```

Comment:
- [ Optional ] The contact section is skewed in mobile version and I believe it is due to the background image for desktop used here for mobile as well. Can we rectify this?
#### Design
![mobile-contact-page-figma](https://user-images.githubusercontent.com/44057863/167739426-87fe8e57-108a-467c-a445-f6f1b0467355.png)
#### Imp
![mobile-contact-page](https://user-images.githubusercontent.com/44057863/167739456-1663682a-08db-474c-a1dd-fba9daeb6e42.png)
l